tests/test_aqmon.py

```diff
@@ -0,0 +1,267 @@
+import os.path
+from unittest import TestCase, skipIf
+import numpy as np
+import strax
+import straxen
+import shutil
+from straxen.test_utils import nt_test_run_id as test_run_id_nT
+
+
+class DummyAqmonHits(strax.Plugin):
+    """
+    Dummy plugin to make some aqmon hits that may span ON and OFF
+    signals over chunks
+
+    There are two channels, that signify ON or OFF of a logic signal
+
+    We are interested in the deadtime, which is the time difference
+    between ON and OFF singals. This plugin computes that deadtime, even
```

Comment:
Okay, but I think this is only the deadtime of the digitizer not the "true" dead time of the detector. For the latter one has to include the daq-veto cut boundaries.

Reply:
right, as discussed, it's the other way around. This is the true deadtime, but the effective livetime loss may be greater and depends on the length of the signals we expect

Reply:
We're getting into semantics here. This plugin computes DAQ deadtime, which is closely related but not identical to detector deadtime.